Тест Recursion

 : Действия
 
  : Действия1
   'B' .
  ;
  
  'A' .
  5 раз Действия1
 ;
 
 // VAR l_WasException
 // false =: l_WasException
 
 // TRY
 10 раз Действия
 // EXCEPT
 //  "Сравнить текущее исключение с эталоном"
 //  true =: l_WasException
 //  DROP
 // - чистим мусорок (цифру 5) на стеке
 // END 
 // l_WasException ?ASSURE 'Почему-то стали поддерживаться рекурсивные вызовы слов'
;

Recursion